app/controllers/groups.js

```diff
@@ -113,15 +113,17 @@ module.exports = function(app) {
     Transaction
       .find({
         attributes: [
-          [sequelize.fn('SUM', sequelize.col('amount')), 'total']
+          [sequelize.fn('SUM', sequelize.col('netAmountInGroupCurrency')), 'total']
```

Comment:
It's a little tricky. There can be up to three currencies:
- DonationCurrency
- TxnCurrency
- GroupCurrency

Ideally, DonationCurrency and GroupCurrency is the same. We have had exceptions like wwcode_df page supports donations in USD and EUR (when Github wanted to donate $1000 USD to them). That meant that DonationCurrency != GroupCurrency.

Since there are two currencies in that table and we are going through migrations on renaming some stuff, I'd prefer to be clear on every amount which currency we are talking about until migration is done.